kool-core/src/commonMain/kotlin/de/fabmax/kool/scene/animation/Animation.kt

```diff
@@ -149,8 +152,8 @@ class AnimatedTransformGroup(val target: Node) : AnimationNode {
         }
 
         t.translate(animTranslation.mul(weight))
-        t.rotate(MutableQuatF().slerp(animRotation, weight))
-        t.scale(MutableVec3f(Vec3f.ONES).lerp(animScale, weight))
+        t.rotate(baseRotation.mix(animRotation, weight))
```

Comment:
Now with `baseRotation` and `baseScale` being members, I think it's also needed to reset them before calling mix, otherwise the previous values will influence the outcome. So:

```
t.rotate(baseRotation.setIdentity().mix(animRotation, weight))
t.scale(baseScale.set(Vec3f.ONES).mix(animScale, weight))
```